src/js/main.js

```diff
@@ -39,7 +65,7 @@ Site = {
   setIsMobileWidth: function() {
     var _this = this;
 
-    if (_this.$window.width() >= _this.mobileThreshold) {
+    if (_this.window.width >= _this.mobileThreshold) {
```

Comment:
yeah its kinda messy. theres `_this.$window` and `_this.window`. `_this.window` stores values such as window size and window center. this specific change is only a micro omprimization